docs/guides/executor.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Executor primitive (beta)
+description: Executor is a lower-level primitive that provides more flexibility when customizing your error mitigation workflow.
+
+---
+
+# Executor primitive (beta)
+
+<Admonition type="note" title="Beta release">
+The Executor primitive is currently in beta and might not be stable. You are invited to test it and provide feedback. (((how?)))
+</Admonition>
+
+The Executor primitive is designed for quantum information scientists to run utility-scale experiments. Executor provides finer control over advanced techniques by shifting the generation of circuit variants to the server side, saving network costs and making the process explicit and customizable. With Executor, users can fine-tune error mitigation and other techniques to find the right balance of speed and quality in their utility-scale workloads.
```

Comment:
```suggestion
The Executor primitive and its supporting tools are designed for quantum information scientists to run utility-scale experiments that require more control and flexibility. Executor provides the ingredients to capture design intents on the client side, and shifts the costly generation of circuit variants to the server side, so that you can fine-tune error mitigation and other techniques without sacrificing performance. Executor's clear, composable model for execution workflows makes it easier to experiment with new techniques, reproduce results, and share methods.
```